self-core-impl/src/main/java/com/selfxdsd/core/managers/Confused.java

```diff
@@ -0,0 +1,39 @@
+package com.selfxdsd.core.managers;
+
+import com.selfxdsd.api.Event;
+import com.selfxdsd.api.pm.Conversation;
+import com.selfxdsd.api.pm.Step;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Conversation where the PM says "didn't understand".
+ * @author criske
+ * @version $Id$
+ * @since 0.0.20
+ */
+public final class Confused implements Conversation {
+
+    /**
+     * Logger.
+     */
+    private static final Logger LOG = LoggerFactory.getLogger(
+        Confused.class
+    );
+
+    @Override
+    public Step start(final Event event) {
+        final Step steps;
+        if (Event.Type.CONFUSED.equals(event.type())) {
+            steps = new SendReply(
+                event.project().language().reply("misunderstand.comment"),
```

Comment:
@criske Can you format this reply to add the comment's author? Like [here](https://github.com/self-xdsd/self-core/blob/master/self-core-impl/src/main/java/com/selfxdsd/core/managers/Hello.java#L64).

String.format(...) will replace the ``%s`` with the specified value.